src/common/local_file_system.cpp

```diff
@@ -284,17 +285,26 @@ bool LocalFileSystem::DirectoryExists(const string &directory) {
 	return false;
 }
 
-bool LocalFileSystem::FileExists(const string &filename) {
+bool LocalFileSystem::FileExists(const string &filename, bool empty_is_valid) {
 	if (!filename.empty()) {
+#if defined(_WIN32) && defined(__MINGW32__)
```

Comment:
Windows (and MingW) use a different code path - this function is never called on Windows.